### ISSUE TITLE
can't have more than 4 segments

### ISSUE THREAD

**Issue by younitee** [lorensr/segmented-control#12]
it seems the CSS breaks if 5 or more items added

**Comment by StructByLightning** [comment]
Still having this problem. To be more specific, it only shows the selected piece. So if I have:

```
                  <SegmentedControl
                    name="oneDisabled"
                    options={[
                      { label: "One", value: "one", disabled: true },
                      { label: "Four", value: "fousr" },
                      { label: "Four", value: "fsour" },
                      { label: "Three", value: "three", default: true },
                      { label: "Four", value: "four" }
                    ]}
                    style={{ width: 800, color: '#ab47bc' }} // purple400
                  />
```

then I get a box with the word "Three" centered in it, but I can't see "One", "Two", or "Four". Pretty critical bug if you ask me.

**Comment by StructByLightning** [comment]
Fixed with #14 .

**Comment by lorensr** [comment]
Thanks, published to npm: https://github.com/lorensr/segmented-control/releases/tag/0.1.10
